minimal.tfvars

```diff
@@ -30,5 +30,7 @@ deploy_titiler            = false
 deploy_stacfastapi        = false
 deploy_swoop_api          = true
 deploy_swoop_caboose      = false
+deploy_db_migration       = false
 deploy_postgres           = true
+deploy_db_init            = true
```

Comment:
Is there any particular reason why `deploy_db_init` is true but `deploy_db_migration` is false?